package.json

```diff
@@ -12,18 +15,31 @@
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
     "lint:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix"
   },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx,ts}": [
+      "eslint --fix",
+      "prettier --write",
+      "npm run test -- --findRelatedTests"
+  ],
```

Comment:
We currently don't have tests, therefore we don't need to run `npm run test` at all. Please remove`"npm run test -- --findRelatedTests"`